src/test/resources/it/pagopa/pn/cucumber/baseFunction/RestApiValidation.feature

```diff
@@ -25,6 +25,7 @@ Feature: verifica funzionamento api rest
     Then si tenta il recupero dal sistema tramite codice IUN dalla PA "Comune_1"
     And l'operazione ha generato un errore con status code "404"
 
+  @restApiValidation
```

Comment:
si potrebbero aggiungere altri test taggati con @restApiValidation (come REST_VALIDATION_3

Reply:
abbiamo aggiunto il tag in altri test come da richiesta